### PR TITLE
New Tx Release v3.0.2

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: packages/tx/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+## 3.0.2 - 2021-02-16
+
+Follow-up release on `v3.0.1` which only partly addressed a **critical** bug. An update is - again - strongly recommended.
+
+- Fixes `tx.isSigned()` always returning true when using the `Tx.fromValuesArray() static constructor, see PR [#1077](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1077)
```
Comment (ryanio) [inline packages/tx/CHANGELOG.md:14]: missing closing backquote 
```suggestion
- Fixes `tx.isSigned()` always returning true when using the `Tx.fromValuesArray()` static constructor, see PR [#1077](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1077)
```

**Review by cgewecke — APPROVED** [review]
LGTM! 💯

**Review by ryanio — APPROVED** [review]
lgtm!

**Comment by holgerd77** [comment]
Just released this as [v3.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.0.2) on npm.

**Pull request merged**